tests/test_utilities.py

```diff
@@ -206,4 +207,170 @@ def test_doc_file(self):
                 A = np.asarray(anneal_schedule)[:,1:3],
                 B = np.asarray(anneal_schedule)[:,2],
                 c = np.asarray(anneal_schedule)[:,3],
-                )
\ No newline at end of file
+                )
+
+
+class TestEnergyScalesCustomSchedule(unittest.TestCase):
+
+    schedule_default = [
+        [ 0. ,  9.3,  0. ,  0.1],
+        [ 0.1,  9. ,  0.5,  0.2],
+        [ 0.2,  8. ,  1.2,  0.4],
+        [ 0.3,  7. ,  1.7,  0.5],
+        [ 0.4,  6. ,  3. ,  0.7],
+        [ 0.5,  5. ,  5. ,  0.8],
+        [ 0.6,  4. ,  7. ,  1. ],
+        [ 0.7,  3. ,  9. ,  2. ],
+        [ 0.8,  2. , 12. ,  3. ],
+        [ 0.9,  1. , 14. ,  4. ],
+        [ 1. ,  0. , 16. ,  8. ]]
+
+    def test_input_verification(self):
+
+        simple_forward_schedule = [
+            [0, 0],
+            [20, 0.3],
+            [100, 1]]
+
+        # Schedules for both, as lists
+        out= energy_scales_custom_schedule(
+            default_schedule=self.schedule_default,
+            custom_schedule=simple_forward_schedule)
+        self.assertTrue(len(out) == len(self.schedule_default))
```

Comment:
`assertEqual`, here and below

Reply:
Closed by [this commit](https://github.com/dwavesystems/dwave-system/pull/570/commits/959cb9ad71cac9a40b18907aafb92450d216fa5d), thanks!